app/controllers/rents_controller.rb

```diff
@@ -0,0 +1,16 @@
+class RentsController < ApplicationController
+
+	def index
+		#active = Rent.connection.quote_column_name('is_active?')
+		@rents = Rent.where(:is_active? == true).where('start <= ?', DateTime.now.beginning_of_day)
```

Comment:
`.where(:is_active? == true)` (which is buggy) and `.where('start <= ?', DateTime.now.beginning_of_day)` are used in multiple places. It should be extracted into `scopes`. Same for other `.where` calls